src/experiments/rl_envs/JuliaRL_SAC_Pendulum.jl

```diff
@@ -58,15 +59,15 @@ function RLCore.Experiment(
             α = 0.2f0,
             batch_size = 64,
             start_steps = 1000,
-            start_policy = RandomPolicy(-1.0..1.0; rng = rng),
+            start_policy = RandomPolicy(Space([-1.0..1.0 for _ in 1:na]); rng = rng),
             update_after = 1000,
             update_every = 1,
             rng = rng,
         ),
         trajectory = CircularArraySARTTrajectory(
             capacity = 10000,
             state = Vector{Float32} => (ns,),
-            action = Float32 => (),
+            action = Float32 => (na,),
```

Comment:
Feels to me like this should be `action = Vector{Float32} => (na,),` but it seems like both work. I'm not really sure how the trajectories are built, will take a look later unless someone has more insight and can tell me.

Reply:
Yes, they are essentially the same. But  `action = Vector{Float32} => (na,),` is a more standard style to align with the doc.